src/content/cloud/okteto-cli.mdx

```diff
@@ -57,36 +58,72 @@ dev:
+- `OKTETO_DOMAIN`: the domain where Okteto exposes your application endpoints.
 - `OKTETO_NAMESPACE`: the namespace where your application is going to be installed.
+- `OKTETO_USERNAME`: your username in Okteto.
+- `OKTETO_REGISTRY_URL`: the url of the [Okteto Registry](cloud/registry.mdx).
 - `OKTETO_GIT_BRANCH`: the name of the Git branch currently being deployed.
 - `OKTETO_GIT_COMMIT`: the SHA1 hash of the last commit of the branch.
-- `OKTETO_REGISTRY_URL`: the url of the [Okteto Registry](cloud/registry.mdx).
-- `BUILDKIT_HOST`: the address of the [Okteto Cloud buildkit instance](cloud/build.mdx). You can use it to build your images with any CLI tool that supports Buildkit builds.
 
-In order to refer to images in the `build` section of the Okteto Manifest, you can use the following environment variables:
-- `${OKTETO_BUILD_<service>_IMAGE}`: the full image reference
-- `${OKTETO_BUILD_<service>_REGISTRY}`: the registry URL where the image was pushed
-- `${OKTETO_BUILD_<service>_REPOSITORY}`: the name of the image that was pushed
-- `${OKTETO_BUILD_<service>_SHA}`: The latest tag and the SHA of the image
+#### Build Environment Variables
 
-For example, for the image `registry.cloud.okteto.net/cindy/hello-world:okteto`, you would have the following values:
+In order to refer to images in the [build](reference/manifest.mdx/#build-object-optional) section of the Okteto Manifest, you can use the following environment variables:
+- `OKTETO_BUILD_<image>_IMAGE`: the full image reference
+- `OKTETO_BUILD_<image>_REGISTRY`: the registry URL where the image was pushed
+- `OKTETO_BUILD_<image>_REPOSITORY`: the name of the image that was pushed
+- `OKTETO_BUILD_<image>_SHA`: The latest tag and the SHA of the image
+
+For example, if the image `hello-world` has built the following image tag`registry.cloud.okteto.net/cindy/hello-world:okteto`, you would have the following values:
 
 - `OKTETO_BUILD_HELLO_WORLD_IMAGE`: registry.cloud.okteto.net/cindy/hello-world@sha256:xxx
 - `OKTETO_BUILD_HELLO_WORLD_REGISTRY`: registry.cloud.okteto.net
 - `OKTETO_BUILD_HELLO_WORLD_REPOSITORY`: cindy/hello-world
 - `OKTETO_BUILD_HELLO_WORLD_SHA`: okteto@sha256:xxx
 
-### Built-in Tools When Deploying To Okteto Cloud
+> if an `<image>` name includes the symbol `-`, in the corresponding `OKTETO_BUILD_<image>` environment variables, the symbol `-` will be replaced by `_`.  For example, if `<image>` is `name-with-hyphen`, the environment variable names will start with `$OKTETO_BUILD_NAME_WITH_HYPHEN_`.
+
+#### Dependencies Environment Variables
+
+In order to refer to variables of [dependencies](reference/manifest.mdx/#dependencies-object-optional) you can use the following environment variables:
+- `OKTETO_DEPENDENCY_<dependency>_VARIABLE_<variable>`: the value of a variable of a dependency.
+
+For example, if you have a dependency to https://github.com/okteto/my-database, and https://github.com/okteto/my-database has the following Okteto Manifest:
```

Comment:
I think we should include a minimalistic (but functional) manifest for both the dependency and the caller of the dependency instead of this description so it's clearer.